examples/jsm/nodes/lights/LightContextNode.js

```diff
@@ -21,7 +21,7 @@ class LightContextNode extends ContextNode {
 
 		const { lightingModelNode } = this;
 
-		this.context.reflectedLight = reflectedLight();
+		this.context.reflectedLight = reflectedLight;
```

Comment:
Share the same `reflectedLight` for all is incompatible with `reflectedLight.add()`

Reply:
Fixed.